crates/sol-macro/src/expand/struct.rs

```diff
@@ -91,8 +91,6 @@ pub(super) fn expand(cx: &ExpCtxt<'_>, s: &ItemStruct) -> Result<TokenStream> {
 
         #[allow(non_camel_case_types, non_snake_case, clippy::style)]
         const _: () = {
-            use ::alloy_sol_types::private::*;
```

Comment:
this failed with Multicall which has a `Result` struct because of import ambiguity, we don't need to do glob imports anymore